ush/machine/noaacloud.yaml

```diff
@@ -5,14 +5,14 @@ platform:
   CCPA_OBS_DIR: /contrib/EPIC/UFS_SRW_App/develop/obs_data/ccpa/proc
   MRMS_OBS_DIR: /contrib/EPIC/UFS_SRW_App/develop/obs_data/mrms/proc
   NDAS_OBS_DIR: /contrib/EPIC/UFS_SRW_App/develop/obs_data/ndas/proc
-  METPLUS_PATH: /contrib/EPIC/spack-stack/envs/ufs-srw-dev.intel/install/intel/2021.3.0/metplus-4.1.0-t4mkmbg
+  METPLUS_PATH: /contrib/EPIC/spack-stack/envs/srw-develop-intel/install/intel/2021.3.0/metplus-4.1.0-4mq7qyb
   MET_BIN_EXEC: bin
-  MET_INSTALL_DIR: /contrib/EPIC/spack-stack/envs/ufs-srw-dev.intel/install/intel/2021.3.0/met-10.1.0-mqwqjxt
+  MET_INSTALL_DIR: /contrib/EPIC/spack-stack/envs/srw-develop-intel/install/intel/2021.3.0/met-10.1.0-7kjlehi
   RUN_CMD_FCST: mpiexec -np ${PE_MEMBER01}
   RUN_CMD_POST: mpiexec -np $nprocs
   RUN_CMD_SERIAL: time
   RUN_CMD_UTILS: mpiexec -np $nprocs
-  PRE_TASK_CMDS: '{ ulimit -s unlimited; ulimit -a; }'
+  PRE_TASK_CMDS: '{ ulimit -s unlimited; ulimit -a; sudo yum -y install time; }'
   BUILD_MOD_FN: wflow_noaacloud
```

Comment:
@ulmononian  Do you know why this line is needed? I wanted to remove it in the past because it doesn't make sense to pass the workflow module as the build module. Not specifying it in the machine file is equivalent to using `BUILD_MOD_FN: build_noaacloud_intel', so it would be nice if you can test removing this line to see if it works.

Reply:
@danielabdi-noaa thanks for taking a look at this! let me look into this and i will get back to you.